common/src/format.rs

```diff
@@ -271,6 +271,8 @@ fn parse_precision(text: &str) -> Result<(Option<usize>, &str), &'static str> {
 }
 
 impl FormatSpec {
+    pub const NOT_IN_RANGE: &str = "%c arg not in range(0x110000)";
```

Comment:
We'd better to define a new enum for this type.

```rust
enum FormatSpecError {
     ...
}

impl IntoPyException for FormatSpecError {
   ...
}
```

This is also a possible fix of the problem, but I'd more like to avoid string comparison to decide the error type from python side.

Reply:
Code fix completed.